src/common/public/oscap.h

```diff
@@ -91,7 +91,8 @@ typedef enum oscap_document_type {
 	OSCAP_DOCUMENT_XCCDF,                 ///< XCCDF benchmark file
 	OSCAP_DOCUMENT_CPE_LANGUAGE,          ///< CPE language file
 	OSCAP_DOCUMENT_CPE_DICTIONARY,        ///< CPE dictionary file
-	OSCAP_DOCUMENT_CVE_FEED,	      ///< CVE NVD feed
+	OSCAP_DOCUMENT_CVE_FEED,              ///< CVE NVD feed
+	OSCAP_DOCUMENT_CVRF_FEED,             ///< CVRF feed
```

Comment:
THIS BREAKS API & ABI, DEFINITELY NEEDS TO BE FIXED. This changes enum values of everything that comes after CVRF feed!